requirements.txt

```diff
@@ -5,4 +5,4 @@ pycoingecko==3.1.0
 pandas==2.0.3
 simplejson==3.19.2
 
-git+https://github.com/BalancerMaxis/bal_addresses@0.8.4
```

Comment:
This needs to be updated to 0.9.1 once released, should remain in draft until then.